queries.sql

```diff
@@ -33,4 +33,30 @@ WHERE NOT name = 'Gabumon';
 -- Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
 SELECT *
 FROM animals
-WHERE weight_kg BETWEEN 10.4 and 17.3;
\ No newline at end of file
+WHERE weight_kg BETWEEN 10.4 and 17.3;
+-- How many animals are there?
+SELECT COUNT(*)
+FROM animals;
+-- How many animals have never tried to escape?
+SELECT COUNT(escape_attempts)
+FROM animals
+WHERE escape_attempts = 0;
+-- What is the average weight of animals?
+SELECT AVG(weight_kg)
+FROM animals;
+-- Who escapes the most, neutered or not neutered animals?
+SELECT AVG(escape_attempts)
+FROM animals
+GROUP BY neutered;
+-- What is the minimum and maximum weight of each type of animal?
+SELECT species,
+  MIN(weight_kg) as MIN_WEIGHT,
+  MAX(weight_kg) as MAX_WEIGHT
+FROM animals
+GROUP BY species;
+-- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
+SELECT species,
+  AVG(escape_attempts) as Average_escape_attempts
+FROM animals
+WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
```

Comment:
- [ ] Please kindly include queries that make `UPDATE` and `DELETE` transactions as specified in the requirements

Reply:
Sure, will do! Thank you. @AnselemOdims